civis/io/_files.py

```diff
@@ -495,10 +508,13 @@ def dataframe_to_file(df, name='data.csv', expires_at='DEFAULT',
     name : str, optional
         The name of the Civis File
     expires_at : str, optional
-        The date and time the file will expire. If not specified, the file will
-        expire in 30 days. To keep a file indefinitely, specify null.
-        If provided, this must be either `None` or
-        a valid RFC3339 date/Time string.
+        The date and time the file will expire. If not specified, the file
+        will expire in 30 days. To keep a file indefinitely, specify ``None``.
+        To specify a date and time, format it by the ISO 8601 standard,
+        e.g., ``"2020-12-31"``, ``"2020-12-31T23:03:40Z"``,
+        and what the ``isoformat()`` method returns from
+        a :class:`datetime.date <datetime.date>`
+        or :class:`datetime.datetime <datetime.datetime>` object.
```

Comment:
I compiled the Sphinx docs locally and verified that the formatting and links would all be good:

![Screen Shot 2021-11-01 at 2 54 20 PM](https://user-images.githubusercontent.com/86482098/139858617-855f210b-98c7-45fc-8ff7-3d8b5ce386e8.png)